tests/e2e/actions.go

```diff
@@ -1751,35 +1751,64 @@ func (tr TestRun) registerRepresentative(
 	wg.Wait()
 }
 
-type registerConsumerRewardDenomAction struct {
-	chain chainID
-	from  validatorID
-	denom string
+type submitChangeRewardDenomsProposalAction struct {
+	chain   chainID
```

Comment:
Could you make the name more descriptive? This is the consumer chain, I assume?

Reply:
I just removed it, only used in a dummy title anyways :)

https://github.com/cosmos/interchain-security/commit/c4e4a6945ec7c58bf87d8bcce8a18848de00748a

It was the chain in which the prop is executed, which is provider

Reply:
Thanks, makes sense!